mllib/src/main/scala/org/apache/spark/mllib/fpm/AssociationRules.scala

```diff
@@ -142,5 +142,13 @@ object AssociationRules {
     def javaConsequent: java.util.List[Item] = {
       consequent.toList.asJava
     }
+
+    /**
+     * @return [antecedent] => [consequent]: [confidence]
```

Comment:
For each method, there should be a summary sentence at the beginning, which shows up in the API doc by default. Other details are hidden by default. I think this is Scala specific. You can find the official guide at http://docs.scala-lang.org/style/scaladoc.html:

~~~
If the documentation of a method is a one line description of
what that method returns, do not repeat it with an @return annotation.
~~~